element_interface/intan_loader.py

```diff
@@ -53,13 +52,13 @@ def read_header(fid):
     (version["major"], version["minor"]) = struct.unpack("<hh", fid.read(4))
     header["version"] = version
 
-    print("")
-    print(
-        "Reading Intan Technologies RHS2000 Data File, Version {}.{}".format(
-            version["major"], version["minor"]
-        )
-    )
-    print("")
+    # print("")
```

Comment:
Remove commented code.  It's in the history if you need it again.

Reply:
> Remove commented code. It's in the history if you need it again.

Meant to remove all of them, but missed that one. Thanks @dimitri-yatsenko